tests/qchem/test_structure.py

```diff
@@ -314,6 +317,61 @@ def test_inconsistent_active_spaces(
         )
 
 
+def mock_get_cids(identifier, namespcae, pcp=None):
```

Comment:
Why we need this function? If I understand `mol_data` correctly, the data is accessed by defining an identifier. Why there is a need to return the CID?

Reply:
The workflow in `mol_data` is to first obtain the CID for the given molecule using any non-CID identifier they provide via `pubchempy.get_cids` method and then using the obtained CID for obtaining the `pubchem.Compound` data via `pubchem.from_cid`. So I am mocking both these methods here.

Reply:
`pubchempy` is only needed to get the CID from on the identifiers?

Reply:
Also, to get the molecular data using that retrieved CID.